lib/app/timer.ex

```diff
@@ -1,9 +1,10 @@
 defmodule App.Timer do
   use Ecto.Schema
   import Ecto.Changeset
-  import Ecto.Query, warn: false
+  # import Ecto.Query
   alias App.Repo
   alias __MODULE__
+  require Logger
 
   schema "timers" do
     field :end, :naive_datetime
```

Comment:
Naming the field `end` is confusing to `Elixir` ...
The the auto-complete in my editor keeps thinking I want to `end` a function when I type it ...
So thinking of renaming it to `stop`: https://github.com/dwyl/app-mvp/issues/114

Reply:
See: https://github.com/dwyl/app-mvp/pull/90/files#r938451241